### PR TITLE
feat: create basic structure for jupyterbook guides

### PR DESCRIPTION
**Pull request by mimotej**
Basic structure for jupyterbook tutorials (as part of https://github.com/operate-first/support/issues/326). Structure is inspired by Operate first website https://www.operate-first.cloud/ .

### CODE REVIEW

File: content/_config.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+title: "Operate-first tutorials"
```
Comment (tumido) [inline content/_config.yml:1]: Let's keep the repo name here:

```suggestion
title: Hitchhiker's Guide to Operate First
```

**Review by tumido — CHANGES_REQUESTED** [review]
This is a good start @mimotej :+1:

Can you please change few things around?

1. Move `content/*` to the repo root ( there's no need to nest all meaningful repository content into a folder).
2. Change the title per suggestion
3. Can you please add/change the README to actually describe the repo purpose? You can leave the current content of the README below (and nest it from H1 to H2 or something)
4. Can you please create a basic `Pipfile` (and a lock file) so people can do just `pipenv install` and have the dev setup ready?
5. Can you please include some "how to" for dev setup? Like how to build it etc?

You can read more on Pipenv here: https://pipenv.pypa.io/en/latest/

**Review by tumido — CHANGES_REQUESTED** [review]
\+ one more thing, can you please add `.gitignore` for files generated by `jypyter-book build`, please? So it doesn't annoy developers and they don't accidentally include generated files in a PR?

File: content/_config.yml (hunk 1 above)
Comment (mimotej) [reply content/_config.yml]: Resolved in latest commit

**Review by tumido — APPROVED** [review]
/lgtm
/approve

Looks great @mimotej, good job! :+1: I think as a scaffold this can be merged now and with next PR you can focus on the content :slightly_smiling_face:

**Pull request merged**